recipes/snp-pipeline/meta.yaml

```diff
@@ -1,11 +1,11 @@
 package:
   name: snp-pipeline
-  version: "0.7.0"
+  version: "1.0.1"
 
 source:
-  fn: snp-pipeline-0.7.0.tar.gz
-  url: https://pypi.python.org/packages/62/6b/5c7c7d1ad8239f2dd051a91e8bc55cd98418df5719d0fd4605f863f71d9a/snp-pipeline-0.7.0.tar.gz
-  md5: 8f5971ca4729364fd22e26e712be862d
+  fn: snp-pipeline-1.0.1.tar.gz
+  url: https://pypi.python.org/packages/9c/86/e2a212735e229b5dd65c7f58f53822e820507bb4f95f870687f50046f99c/snp-pipeline-1.0.1.tar.gz
+  md5: d42bfefd2ccccdb5f880f3a74f4ff165
 
 build:
   preserve_egg_dir: True
```

Comment:
Please also add to the `build:` section:
```yaml
  entry_points:
    - cfsan_snp_pipeline = snppipeline.cfsan_snp_pipeline:main
    - qarrayrun = snppipeline.qarrayrun:main
```
That way we get scripts created by `conda`/`conda-build` instead of `setuptools`, which should be preferred.